download-libs.sh

```diff
@@ -42,6 +48,7 @@ if [ ! -d closure-compiler/.git ]; then
   fi
   git clone --depth 1 https://github.com/google/closure-compiler closure-compiler
```

Comment:
Ooh, thanks Drew, good to know about the closure compiler branch naming. BTW, why does Google E2E clone instead of using git submodules?